tests/unit/test_loader.py

```diff
@@ -20,23 +22,22 @@ def test_prepare_database(self):
             implied_views = set(f.schema_type_plural for f in vx_bundle.files if f.normalizable)
             self.assertEqual(result & implied_views, implied_views)
 
-    @unittest.skip("WIP")
     def test_insert_into_database(self):
-        self.loader.load(vx_bundle, dict(a='b'))
-
-        with self.db.transaction() as (cursor, tables):
-            # bundle insertion
-            result = tables.bundles.select(vx_bundle.uuid, vx_bundle.version)
-            self.assertIsNotNone(result)
-
-            # join table
-            result = tables.bundles_files.select_bundle(vx_bundle.uuid, vx_bundle.version)
-            self.assertTrue(len(result) > 0)
-
-            # files and view tables
-            for file in vx_bundle.files:
-                result = tables.files.select(file.uuid, file.version)
-                self.assertIsNotNone(result)
+        config.reset_db_session()
+        files = []
+        for f in json.loads(vx_bundle_manifest)['files']:
+            if f["content-type"] == "application/json":
+                files.append(dict(name=f["name"],
+                                  uuid=f["uuid"],
+                                  version=f["version"],
+                                  body=json.loads(load_fixture(f["name"])),
+                                  content_type=f["content-type"],
+                                  size=f["size"]))
+        load_bundle(dict(uuid=vx_bundle_uuid,
```

Comment:
can you test that what is in the db after calling the load function matches what you expect to be in the db?

Reply:
Great point, added a readback test. Please take another look.